packages/bella/src/front-side-dart.js

```diff
@@ -21,7 +21,7 @@ export default function (part) {
   points.hpsCp2 = new Point(points.hps.x, points.cfNeck.y / 2)
   let slope = measurements.shoulderSlope * (2 - options.shoulderSlopeBack) * -1
   let xShoulder =
-    ((measurements.shoulderToShoulder * options.shoulderToShoulderCorrection) / 2) *
+    ((measurements.shoulderToShoulder * (1+options.shoulderToShoulderEase)) / 2) *
```

Comment:
```suggestion
    ((measurements.shoulderToShoulder * (1 + options.shoulderToShoulderEase)) / 2) *
```

Reply:
This seems like Nick fixed a bug here. Wouter?

Reply:
That's just a couple of extra spaces (as `prettier` will overrule, or not, anyway)
Or am I missing something?